<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Журнал проверок</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Даты</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>С</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="begin_date">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>По</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="end_date">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="select_dates">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Внешний вид</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="split_view">
           <property name="text">
            <string>Раздельный</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="join_view">
           <property name="text">
            <string>Объединенный</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="open_instruction">
        <property name="text">
         <string>Инструкция</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="authorization">
          <property name="text">
           <string>Авторизация</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_inspection">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit_app">
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_user_settings"/>
    <addaction name="action_change_password"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_reports">
    <property name="title">
     <string>Отчеты</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_inspectors">
    <property name="title">
     <string>Проверяющий</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_responsible">
    <property name="title">
     <string>Ответственный</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_inspectors"/>
   <addaction name="menu_responsible"/>
   <addaction name="menu_reports"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_change_password">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сменить пароль</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>В разработке</string>
   </property>
  </action>
  <action name="action_user_settings">
   <property name="text">
    <string>Управление пользователями</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
